packages/fxa-settings/src/lib/auth-errors/auth-errors.ts

```diff
@@ -56,8 +56,7 @@ const ERRORS = {
   },
   INVALID_TOKEN: {
     errno: 110,
-    message: 'Invalid session. Please sign out and sign in again.',
-    version: 2,
+    message: 'Invalid session',
```

Comment:
Version 1 was "invalid token" - should this be version 3? I do think "invalid session" is clearer than "invalid token"